script/release.js

```diff
@@ -36,7 +36,7 @@ const release = async () => {
   spawnOrFail('git', ['fetch origin'], { skipOutput: true });
   const currentBranch = (spawnOrFail('git', [' branch --show-current'], { skipOutput: true })).trim();
   const remoteBranch = getCurrentRemoteBranch();
-  if (!remoteBranch || (remoteBranch !== 'origin/main' && (/^origin\/release-[0-9]+\.x$/).test(remoteBranch))) {
+  if (!remoteBranch || (remoteBranch !== 'origin/main' && !(/^origin\/release-[0-9]+\.x$/).test(remoteBranch))) {
```

Comment:
Should this regex check now for `release-${currentVersion}`?

Reply:
No this check is just to make sure we run this from either `main` or `release-2.x` branch (the origin branch we want to cut a release branch from). `release-${currentVersion}` is the destination branch.

Reply:
Got it, thanks!